### PR TITLE
TEST/EXAMPLES: fix uct_hello_world bcopy/zcopy

### PR DESCRIPTION
**Pull request by evgeny-leksikov**
Fixes https://github.com/openucx/ucx/issues/2817

### CODE REVIEW

**Comment by shamisp** [comment]
@keisukefukuda FYI

**Comment by shamisp** [comment]
👍

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7523/ for details (Mellanox internal link).

**Comment by evgeny-leksikov** [comment]
Not related failure https://github.com/openucx/ucx/issues/2822
bot:mlx:retest

**Comment by evgeny-leksikov** [comment]
@keisukefukuda pls try updated patch

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7525/ for details (Mellanox internal link).

File: test/examples/uct_hello_world.c [hunk 1]
```diff
@@ -596,7 +603,7 @@ int main(int argc, char **argv)
         if (cmd_args.func_am_type == FUNC_AM_SHORT) {
             status = do_am_short(ep, id, &cmd_args, str);
         } else if (cmd_args.func_am_type == FUNC_AM_BCOPY) {
-            status = do_am_bcopy(ep, id, &cmd_args, str);
+            status = do_am_bcopy(&if_info, ep, id, &cmd_args, str);
```
Comment (yosefe) [inline test/examples/uct_hello_world.c]: do we need to fix am_short too?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7539/ for details (Mellanox internal link).

**Pull request merged**